.travis.yml

```diff
@@ -14,7 +14,7 @@ cache:
     - "server/node_modules"
 before_script:
     - psql -c 'create database bankaa;' -U postgres
-    - npm run test-db    
+   # - npm run test-db    
```

Comment:
what's the # for?

Reply:
to comment out the line. I found out while it builds on Travis it runs the command twice

Reply:
should I remove the line and commit again?

Reply:
yeah... remove it